README.md

```diff
@@ -44,19 +43,19 @@ This setting, with at least one environment, is **required** for django-pinpayme
 ```python
     PIN_ENVIRONMENTS = {
         'test': {
-            'key': 'pk_qokBvPpEHIVmNETSoSdDVYP',
-            'secret': 'MBjZMurpDtjDANDNFQObZmBhMg',
-            'host': 'test-api.pin.net.au',
+            'key': '',
```

Comment:
I disagree with removing these sample details. Showing a sample of "pk_xxxx" for the public key, and a secret wtihout a pk_ prefix, should help users figure out which token goes where. Additionally, there are only ever two hosts that will be used: test-api, or api. There is no reason to exclude these from our documentation as it will lead to users needing to guess.